custom_components/hubspace/light.py

```diff
@@ -68,8 +68,17 @@ def _convert_color_temp(value):
     return 1000000 // int(value)
 
 
-def _add_entity(entities, hs, model, deviceClass, friendlyName, debug):
+def create_entity(hs: HubSpace, model: str, deviceClass: str, friendlyName: str, debug: bool) -> list:
```

Comment:
This function previously took in the list and returned the same memory reference. The new function provides a list of entities that should be added, and its the drives responsibility to extend the list of everything that must be added.